src/panels/logbook/ha-logbook.ts

```diff
@@ -208,15 +209,30 @@ export class HaLogbook extends LitElement {
     );
   }
 
-  private async _unsubscribe(): Promise<void> {
-    if (this._subscribed) {
+  /**
+   * Unsubscribe from a logbook stream since
+   * - we are unloading the page
+   * - we are about to resubscribe
+   * - the entity is not being tracked in the logbook
+   *   and will not return results ever
+   * - the requested start time is in the future
+   *
+   * In cases where no events are expected, we set this._logbookEntries
+   * to an empty list to show a no results message.
+   *
+   * @param loading Indicates if the page should be put in a loading state again.
+   */
+  private _unsubscribe(loading: boolean): void {
+    if (this._unsubLogbook) {
       try {
-        await this._subscribed();
-        this._subscribed = undefined;
-        this._pendingStreamMessages = [];
+        this._unsubLogbook.then((unsub) => unsub());
```

Comment:
This will not throw as it is not awaited, if you want to do something with the error, either use:

```ts
try {
	const unsub = await this._unsubLogbook;
	unsub();
} catch (...)
{ .. }
```

OR:

```ts
this._unsubLogbook.then((unsub) => unsub()).catch(err => {console.error("Error unsubscribing:", err)});
```

I this case I would go for the second (although I don't think we have to log) as we dont have to wait for the unsub I think to continue?

Reply:
The purpose of this code was clean up the variables, so the filters around this components remain to work, so the try catch finally can be removed then.